### PR TITLE
Fix typos in descriptions of server source file

### PR DESCRIPTION
**Pull request by asdofindia**
I used hunspell backend with linter-spell plugin of atom to find these typos. There are several capitalizations issues that I've left untouched (pop3, POP3, etc)

### CODE REVIEW

File: config/locales/server.en.yml [hunk 1]
```diff
@@ -1128,7 +1128,7 @@ en:
     body_min_entropy: "The minimum entropy (unique characters, non-english count for more) required for a post body."
     allow_uppercase_posts: "Allow all caps in a topic title or a post body."
 
-    title_fancy_entities: "Convert common ASCII characters to fancy HTML entities in topic titles, ala SmartyPants http://daringfireball.net/projects/smartypants/"
```
Comment (ZogStriP) [inline config/locales/server.en.yml]: `ala` is right here.

Comment (asdofindia) [inline config/locales/server.en.yml]: Okay, let me amend the commit then. But, what does "ala" mean here?

Comment (asdofindia) [inline config/locales/server.en.yml]: Ah, "ala" as a colloquial form of "[a la](https://en.wiktionary.org/wiki/a_la#Alternative_forms)". Hmm.

**Comment by coding-horror** [comment]
I reviewed and these look good -- thanks!

**Pull request merged**